homeassistant/components/synology/camera.py

```diff
@@ -87,12 +105,17 @@ def camera_image(self):
 
         return await async_aiohttp_proxy_web(self.hass, request, stream_coro)
 
+
     @property
     def name(self):
         """Return the name of this device."""
         return self._camera.name
 
     @property
+    def home_mode(self):
```

Comment:
It makes no sense to have this be a property of cameras. Please remove this and instead make it a binary sensor.